### ISSUE TITLE
Update the project for OWASP Top 10 for 2017 once it is released.

### ISSUE THREAD

**Issue by ckarande** [OWASP/NodeGoat#99]

**Comment by jboyer2012** [comment]
I'm already looking a good bit into the OWASP Top 10 for 2017 for [my blog](https://greenmachinesec.wordpress.com/2017/05/03/breaking-down-the-owasp-top-10-2017-rc-part-1-numbers-one-through-five/). I' like to take this on and see what I can add. Would you prefer a separate branch for this work?

**Comment by binarymist** [comment]
Yes please @jboyer2012 and please make sure to follow: https://github.com/OWASP/NodeGoat/#ways-to-contribute, and also run the security regression test before and after your work: https://github.com/OWASP/NodeGoat/wiki/NodeGoat-Security-Regression-tests-with-ZAP-API
